packages/web/src/components/gcds-link/stories/gcds-link.stories.js

```diff
@@ -0,0 +1,216 @@
+export default {
+  title: 'Components/Link',
+
+  parameters: {
+    actions: {
+      argTypesRegex: '^on.*',
+      handles: ['click', 'change', 'focus', 'blur'],
+    },
+  },
+
+  argTypes: {
+    // Props
+    size: {
+      control: { type: 'radio' },
+      options: ['regular', 'small'],
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: 'regular' },
+      },
+    },
+    display: {
+      control: 'select',
+      options: ['inline', 'inline-block', 'block'],
+    },
+    href: {
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+    rel: {
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+    target: {
+      control: 'select',
+      options: ['_blank', '_self', '_parent', '_top'],
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+    external: {
+      control: 'boolean',
+      table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: false },
+      },
+    },
+    download: {
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+    type: {
+      control: { type: 'text' },
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: 'button' },
```

Comment:
We don't seem to have a default value defined for the type, can you update the default value her to `-`?